src/schedule-and-details/credit-section/CreditSection.scss

```diff
@@ -0,0 +1,9 @@
+.credit-section {
+  .credit-info-list {
```

Comment:
Do we need additional nesting?

```
.credit-section .credit-info-list {
  // your code
}
```

Reply:
I use the approach that I used before with a few classes in the styles, but I've changed